samples/net/wifi/Kconfig.sysbuild

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config WIFI_NRF70
+	default y if BOARD_NRF7002DK_NRF5340_CPUAPP || \
+		     BOARD_NRF7002DK_NRF5340_CPUAPP_NS || \
+		     BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001 || \
+		     BOARD_NRF7002DK_NRF5340_CPUAPP_NRF7001_NS
```

Comment:
no

Reply:
This was cherry-picked from https://github.com/nrfconnect/sdk-zephyr/pull/2072 I think I understand your concern that for customer boards the sample might not build, but I don't see an alternative?

Reply:
You set it in samples that need it, just like every other feature in zephyr and ncs does

Reply:
Yes, this is the in the sample itself, am I missing anything?

Reply:
Seems I mistakenly thought it was in the sysbuild folder, my bad